test/linalg/triangular.jl

```diff
@@ -245,10 +250,12 @@ for elty1 in (Float32, Float64, Complex64, Complex128, BigFloat, Int)
 
                 # Triangular-Triangular multiplication and division
                 elty1 != BigFloat && elty2 != BigFloat && @test_approx_eq full(A1*A2) full(A1)*full(A2)
-                @test_approx_eq full(A1'A2) full(A1)'full(A2)
-                @test_approx_eq full(A1*A2') full(A1)*full(A2)'
-                @test_approx_eq full(A1'A2') full(A1)'full(A2)'
-                @test_approx_eq full(A1/A2) full(A1)/full(A2)
+                if !(elty1 == Int && elty2 == Int)
```

Comment:
if you remove the rule about the `Int`s you'll get an `InexactError` for those types with `debug = true`